.github/workflows/example.yml

```diff
@@ -0,0 +1,23 @@
+name: Example Harness Deploy
```

Comment:
delete, move to the readme

Reply:
👍 move to Readme, so it's in the docs front and center when you hit the repo, or change filename so it's ignored harness-deploy.yml.example for example :)

Reply:
It is, I already added the Usage docs. Will remove the example.